### ISSUE TITLE
wl doesn't handle multiple accounts on same host

### ISSUE THREAD

**Issue by levitte** [wanderlust/wanderlust#163]
```
%:account1/login@myhost.se:993! /
%:account2/login@myhost.se:993! /
```

The first account displays fine, the other doesn't.  When trying to open it, I get this message (from `*Messages*`):

```
Auto plugged off at myhost.se:993 :elmo-network-initialize-session
elmo-network-open-session: Cannot open: elmo-network-initialize-session
```

**Comment by egh** [comment]
Works for me, perhaps there is some other issue?
![image](https://user-images.githubusercontent.com/22718/56852050-dd7ab600-68ca-11e9-8077-36648b767a20.png)

**Comment by egh** [comment]
Also, try `M-t M-t` to toggle on/off all connections.

**Comment by levitte** [comment]
Hmmmm, if I creep through the session initialization with the debugger, it works...  so there could be a timing issue?  Dunno, it's a wild guess.

Is there a way to view the actual IMAP conversation between wanderlust and the server?

**Comment by dmj** [comment]
(setq elmo-imap4-debug t) and then there is a *IMAP4 Debug* buffer.

**Comment by levitte** [comment]
Ok, the stuff that fails doesn't even register in that buffer!

I just looked into the `elmo-network-initialize-session` method in `elmo-imap4.el`, and found this:

https://github.com/wanderlust/wanderlust/blob/6528a0032342ec7bf6a95dda9b60d67292eaadfc/elmo/elmo-imap4.el#L1124-L1128

Starting to think I should uncomment that code and see where it gets me...

**Comment by egh** [comment]
Possibly completely unrelated, but see https://github.com/magit/ghub/issues/81

**Comment by levitte** [comment]
Not quite related...  but it turned out that I had an invalid certificate, and when I was asked if I wanted to accept just for this session or always, I answered for this session...  I just tried "always", and the problem when \*poof\*, gone.  So it turns out not to be a wl problem, but something underlying, some gnutls session management, I guess?

Thanks for your patience while I wade through these problems.
